uiowa/uiowa
# [Registrar] Added css override for label

Resolves https://github.com/uiowa/uiowa/issues/7909. 

# How to test

```
ddev blt ds --site=registrar.uiowa.edu && ddev drush @registrar.local uli /credential-validation
```

Confirm that "Please enter CeDID" label is bigger.
